torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -264,11 +326,29 @@ struct SourceImporterImpl : public Resolver,
       // ClassTypes)
       return importNamedTuple(qualified_name, class_def);
     } else if (superclass_name == "Interface") {
+      // Parse the set of ignored arguments for each method from the ClassDef.
+      auto ignored_argument_names = getIgnoredArgumentNames(class_def);
+      // Strip the Assign for the ignored argument names from the ClassDef for
+      // the interface because define_interface doesn't handle those.
+      ClassDef stripped_class_def = stripNonDefFromClass(class_def);
```

Comment:
This is not elegant but is a necessary consequence of the fact that `ClassDef` is used to store the AST for interface types and that the parsing machinery for class types is reused for interfaces.